target_chains/cosmwasm/tools/src/pyth-wrapper.ts

```diff
@@ -0,0 +1,118 @@
+import {
+  ChainExecutor,
+  ExecuteContractResponse,
+} from "./chains-manager/chain-executor";
+import { ChainQuerier } from "./chains-manager/chain-querier";
+
+export type UpdateFeeResponse = {
+  denom: string;
+  amount: string;
+};
+
+export type Price = {
+  price: number;
+  conf: number;
+  expo: number;
+  publish_time: number;
+};
+
+export type PriceFeedResponse = {
+  id: string;
+  price: Price;
+  ema_price: Price;
+};
+
+export class PythWrapperQuerier {
+  constructor(private chainQuerier: ChainQuerier) {}
+
+  // get update fee
```

Comment:
this comment isn't very helpful -- i get the same information from the method name itself. You should document what it does and how to call it. E.g.:

"Get the fee required to update the Pyth on-chain contract with the given VAAs. This function will invoke the corresponding function on the contract itself and return the result. The result is represented as both a quantity of tokens and a denomination which may vary depending on the chain being queried."